### DISCUSSION TITLE
Problemas con el compilador

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#27] [Preguntas y respuestas] [answered]
**Hola!** estuve haciendo los ejercicios del Tp1 y cuando intentaba correr el programa con el compilador este no funcionaba. 

![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134400606/89879507-f41b-44af-aa2c-f2fae4683199)

se que hay algo que estoy haciendo mal pero no estoy seguro de que. Tengo que usar la ubicacion directa del programa para correrlo?

**Comment by JeanFranco99** [comment]
Si no me equivoco, te falta poner .exe 
`./prueba.exe` debería funcionar
En caso de que no, fíjate de hacer 
`gcc ejercicio2.c`
`./a.exe`
Y ahí debería funcionar
Pero para mí (puede que me equivoqué) te falta la extensión `.exe` para ejecutarlo correctamente

**Reply by rinrot** [reply]
![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134400606/193946ca-f07d-40a5-b83b-3a33f441a54f)

![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134400606/b0418cfa-efce-4acb-8773-5e6b55870bef)

mismo resultado. me pone que el "." no se reconoce como comando nomas

**Reply by JeanFranco99** [reply]
Hiciste el ejercicio 1? Tuviste el mismo problema?

**Reply by JeanFranco99** [reply]
Fíjate de abrir el git bash (botón derecho dentro de la carpeta, abrir git bash en esta dirección) y probar otra vez

**Comment by martinvilu** [comment, marked as answer]
En la consola que estás usando, no hace falta el `./`, solo meté el nombre del `.exe`.

Ah, y esa parte es opcional :-); por lo que podés hacer directamente `$>a` que debiera de funcionar.

También, tenés con la tecla de tabulador, para completar nombres; funciona mejor si ya tenes alguna letra escrita.

**Reply by rinrot** [reply]
ahora si funca!! gracias por responder profe!
